src/bidiMapper/domains/context/cdpTarget.ts

```diff
@@ -115,6 +124,9 @@ export class CdpTarget {
     });
 
     await this.#cdpClient.sendCommand('Runtime.runIfWaitingForDebugger');
+
+    await this.loadTopLevelPreloadScripts();
```

Comment:
We want to setup preload scripts BEFORE the target started living it's own life.
```suggestion
    await this.loadTopLevelPreloadScripts();

    await this.#cdpClient.sendCommand('Runtime.runIfWaitingForDebugger');
```